epoxy-processor/src/main/java/com/airbnb/epoxy/ControllerProcessor.java

```diff
@@ -255,6 +255,8 @@ private void generateHelperClassForController(ControllerClassInfo controllerInfo
           .addMethod(buildSaveModelsForNextValidationMethod(controllerInfo));
     }
 
+    builder.addOriginatingElement(controllerInfo.getControllerClassElement());
```

Comment:
In its current form `ControllerProcessor` relies on `EpoxyProcessor` passing it information about all the models (https://github.com/airbnb/epoxy/blob/master/epoxy-processor/src/main/java/com/airbnb/epoxy/ControllerProcessor.java#L91). Wouldn't this make it aggregating rather than isolating?

Reply:
I don't think so because all the models are fields/properties of the same controller class right? It only cares about their declaration within the controller, not about any of their implementation details.